cruise-control/src/yaml/responses/operationProgress.yaml

```diff
@@ -0,0 +1,18 @@
+OperationProgress:
+  type: object
+  required:
+    - version
+    - operation
+    - operationProgress
+  properties:
+    version:
+      type: integer
+      format: int32
+    operation:
+      type: string
+      enum:
```

Comment:
The operation name is customized(i.e. depend on [what is passed in to `OperationProgress` object](https://github.com/linkedin/cruise-control/blob/c64d73c3c133dcdd705488b9f42ea5e10d34e3bf/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java#L38), can we remove the `enum` here?